shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -989,23 +998,28 @@ - (void)onPreferredStatusBarStyleUpdated:(NSNotification*)notification {
   return [_engine.get() platformViewsController];
 }
 
+#pragma mark - FlutterBinaryMessengerContainer
+- (NSObject<FlutterBinaryMessenger>*)binaryMessenger {
```

Comment:
I'd deprecate this right now as we mentioned and encourage the vc.engine.binaryMessenger path like we mentioned.

Reply:
I can't actually deprecate this =(.  The accessibility bridge requires the indirection to behave correctly:  https://github.com/flutter/engine/blob/185087a65f5104710399d107a5228636b558727c/shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm#L575

Reply:
Done.  I just remembered why it had this dependency and removed it, it shouldn't be needed anymore (and especially after we remove the leak completely).